core/chains/evm/client/pool.go

```diff
@@ -27,6 +27,8 @@ var (
 		Name: "evm_pool_rpc_node_states",
 		Help: "The number of RPC nodes currently in the given state for the given chain",
 	}, []string{"evmChainID", "state"})
+	// NotAcceptedSendOnlyChainID holds the value of the chain ID that gets ignored for sendonly nodes
+	NotAcceptedSendOnlyChainID = big.NewInt(0)
```

Comment:
A global exported variable seems unecessary here. What about just declaring a `zero := big.NewInt(0)` before the for loop to reuse?